src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -192,10 +192,10 @@
         WordEmbeddings wrap different embedding models, such as GloVe. Users can specify which embedding to use. 
         The available options are various versions of <a href="https://nlp.stanford.edu/projects/glove/">GloVe Models</a>, <a href="https://en.wikipedia.org/wiki/FastText">fastText</a>, and <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">SSWE</a>.
         <para>
-          Note: As WordEmbedding requires a column with text vector, e.g. %3C%27this%27, %27is%27, %27good%27%3E, users need to create an input column by
-          using the output_tokens=True for TextTransform to convert a column with sentences like "This is good" into %3C%27this%27, %27is%27, %27good%27 %3E.
-          The suffix of %27_TransformedText%27 is added to the original column name to create the output token column. For instance if the input column is %27body%27,
-          the output tokens column is named %27body_TransformedText%27.
+          Note: As WordEmbedding requires a column with text vector, e.g. &lt;&apos;this&apos;, &apos;is&apos;, &apos;good&apos;&gt;, users need to create an input column by
+          using the output_tokens=True for TextTransform to convert a column with sentences like "This is good" into &lt;&apos;this&apos;, &apos;is&apos;, &apos;good&apos; &gt;.
```

Comment:
What if some one does not want to use TextTransform for creating vector of text. I would also recommend mentioning of wordtokenizer transform here.

Reply:
Not sure if its related to your change...:)